standards/practices/maintaining.md

```diff
@@ -0,0 +1,24 @@
+# Maintaining
+
+## Summary
+We maintain our code and keep it updated whenever Highland is working on it.
```

Comment:
"keep it updated" feels a little vague. Do we want this to mean anything other than keep dependencies updated?

Should we say anything about latest major release or LTS or anything like that? Even if just to say it's a per-project decision that doesn't need to be explicit until it matters enough that it's causing pain?